wagtail/wagtailcore/migrations/0028_merge.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.1 on 2016-02-03 13:50
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('wagtailcore', '0024_alter_page_content_type_on_delete_behaviour'),
+        ('wagtailcore', '0027_fix_collection_path_collation'),
+    ]
+
+    operations = [
+    ]
```

Comment:
Django requires one of these merge migrations (created with `./manage.py makemigrations --merge`) whenever there are migrations with overlapping sequence numbers, to prevent conflicts and to ensure that any later migrations are based on a fully up-to-date version of the database state.

Normally we'd renumber migrations when we merge the PR, to avoid the need for merge migrations - but in this case there's already one real-world site (or maybe more...) using an early version of collections with Wagtail 1.3, so we need to keep the numbers unchanged to avoid upgrade problems.